lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -315,12 +327,15 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
       stringLeaf.append(iter->getSecond());
     }
 
-    // Compute the reference attribute.  If the path is size 1,
-    // then this is just an InnerRefAttr (module--component pair).  Otherwise,
-    // we need to use the symbol of a HierPathOp that stores the path.
-    if (refSendPath.size() == 1)
-      ref = refSendPath.front();
-    else if (!refSendPath.empty())
+    if (refSendPath.empty()) {
+      if (refVal.getDefiningOp())
+        refVal.getDefiningOp()->getParentOp()->dump();
```

Comment:
This looks like debugging code, please remove.

Reply:
Stupid debugging code.